web/src/views/FindSpeaker.vue

```diff
@@ -21,6 +21,17 @@
             {{ $t('findSpeaker.title') }}
           </h2>
         </v-row>
+        <v-row
+          v-if="!speakers.length"
```

Comment:
I think for now this condition is fine, but once the [pagination PR](https://github.com/WWCodeTokyo/speak-her-db/pull/166) is merged, maybe the condition should be changed a little bit so it shows on every pagination event 🤔

Reply:
Great, can you make this update?

Reply:
we need a new PR for moving the pagination into it's own component anyways right? how about fixing it together in that PR? :)

Reply:
either way is fine for me though, just let me know which you prefer :)